### PR TITLE
Okx replacing safeValue to safeBool/Dict/List

### PR DESCRIPTION
**Pull request by rayBastard**

### CODE REVIEW

File: ts/src/okx.ts [hunk 1]
```diff
@@ -2290,20 +2290,12 @@ export default class okx extends Exchange {
         return this.filterBySymbolSinceLimit (sorted, market['symbol'], since, limit) as FundingRateHistory[];
     }
 
-    parseBalanceByType (type, response) {
```
Comment (carlosmiei) [inline ts/src/okx.ts:2293]: why are we removing this?

Comment (carlosmiei) [reply ts/src/okx.ts:2293]: I don't even know how the build is working without this method 🤔

Comment (rayBastard) [reply ts/src/okx.ts:2293]: With no reason. It was my deleting? If it was it is just a typo, although I don't remember removing this method

Comment (rayBastard) [reply ts/src/okx.ts]: I add it

File: ts/src/okx.ts [hunk 2]
```diff
@@ -1623,7 +1623,7 @@ export default class okx extends Exchange {
         //        "msg": ""
         //    }
         //
-        const data = this.safeValue (response, 'data', []);
+        const data = this.safeDict (response, 'data', []);
```
Comment (carlosmiei) [inline ts/src/okx.ts:1626]: @rayBastard the defaultType [] does not match the `safeDict` usage

Comment (rayBastard) [reply ts/src/okx.ts:1626]: Yeah, that should be `safeList`

Comment (rayBastard) [reply ts/src/okx.ts]: Update it

**Pull request merged**